lib/zendesk_apps_tools/server.rb

```diff
@@ -17,6 +17,7 @@ class Server < Sinatra::Base
           last_mtime = curr_mtime
         end
       end
+      settings.parameters.merge!(locale: params['locale'])
```

Comment:
@liulikun we could add an optional parameter to the end of the method signature for `ZendeskAppsSupport::Package#initialize` if that seems better?